components/authorization/cluster/kcp-spi-namespace-admin.yaml

```diff
@@ -10,3 +10,16 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: admin
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kcp-1ry9c9jzxyzk-admin
+  namespace: kcp-1ry9c9jzxyzk
+subjects:
+  - kind: User
+    name: skabashnyuk
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
```

Comment:
just out of curiosity - why do you need that? All modifications in this namespace should happen via the syncer.

Reply:
I fully understand that it's a huge black box, but this is exactly the reason why I would love to not edit anything in these namespaces (even accidentally). We just don't know what we could break there.